javaparser-symbol-solver-testing/src/test/test_sourcecode/javasymbolsolver_0_6_0/src/java-symbol-solver-core/com/github/javaparser/symbolsolver/reflectionmodel/ReflectionParameterDeclaration.java

```diff
@@ -38,7 +38,7 @@ public ReflectionParameterDeclaration(Class<?> type, java.lang.reflect.Type gene
 
     @Override
     public String getName() {
-        throw new UnsupportedOperationException();
+        throw new UnsupportedOperationException("Unable to get the name");
```

Comment:
It would be helpful to include some reasoning here. 

With the reflection model, it is often the case that names are not available.